set.mm

```diff
@@ -621580,6 +621617,15 @@ fixed reference functional determined by this vector (corresponding to
       ( wi wo jao syl6c ) ABCDHEDHCEIDHFGCDEJK $.
   $}
 
+  ${
+    $d x y $.  $d x ph $.
+    spimevw.1 $e |- ( x = y -> ( ph -> ps ) ) $.
+    $( Version of ~ spimefv with ` x ` , ` ph ` disjoint, using only Tarski's
+       FOL axiom schemes.  (Contributed by SN, 21-Sep-2023.) $)
+    spimevw $p |- ( ph -> E. x ps ) $=
+      ( ax-5 spimeh ) ABCDACFEG $.
```

Comment:
That's bj-spimevw. No worries for the tag, but can you add to the comment
> Existential introduction, using implicit substitution. This is to spimeh what spimvw is to spimw.

and then deleted the one in my mathbox.